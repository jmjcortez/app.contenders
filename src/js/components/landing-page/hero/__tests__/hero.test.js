import React from 'react';
import Hero from '../hero.component';
import { shallow } from 'enzyme';

describe('Hero test', () => {
    it('should render correctly with no props', () => {
        const component = shallow(<Hero />);
        
        expect(component).toMatchSnapshot();
    });
});