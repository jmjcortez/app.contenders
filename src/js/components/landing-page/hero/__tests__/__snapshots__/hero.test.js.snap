// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hero test should render correctly with no props 1`] = `
<styled.div>
  <styled.h1>
    Be the best
  </styled.h1>
  <styled.span />
  <styled.div>
    Punch, kick, grapple whatever you wanna do you can do here. Meet awesome people and be the best version of yourself
  </styled.div>
  <styled.div>
    <Styled(Button)
      href="register"
      size="lg"
    >
      Register
    </Styled(Button)>
    <styled.p>
      Discover the experience
    </styled.p>
  </styled.div>
</styled.div>
`;
